scrapers/teiid.py

```diff
@@ -94,6 +96,17 @@ def query(self, sql, db='public', retry=None):
         if retry is not None and retry < 1:
             raise ValueError('The retry keyword must contain a value greater than 0')
 
+        one_min_ago = datetime.utcnow() - timedelta(minutes=1)
+        clean_throttle_tracker = [query_date for query_date in self._query_dates
+                                  if query_date >= one_min_ago]
+        # Replace self._query_dates with only query dates that are from the last minute
+        self._query_dates = clean_throttle_tracker
+        if len(self._query_dates) >= 20:
+            log.info('There have been 20 or more queries in the past minute, let\'s give Teiid a '
+                     '5 second break.')
+            sleep(5)
```

Comment:
I'm wondering what the maximum number of queries we want each scraper to do per minute. Perhaps 5 seconds is too little of a delay.